lib/messages/create/create-issue-attachment.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (issue) => {
```

Comment:
Instead of this, could we re-use the existing message we have? https://github.com/integrations/slack/blob/master/lib/messages/issue.js